media-picker/build.gradle

```diff
@@ -23,19 +23,19 @@ android {
 dependencies {
 
     //noinspection GradleDynamicVersion
-    provided 'com.android.support:appcompat-v7:25+'
+    compileOnly 'com.android.support:appcompat-v7:25+'
```

Comment:
maybe update the appcompat deps to be the same across projects by making a param similar to what we do for minSdkVersion